utils/lua_style_checker.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+#
+
+# Utility to enforce optional parentheses around i18n functions in lua scripts.
+
+import os
+import re
+import sys
+from file_utils import read_text_file, find_files
+
+
+def check_line(filename, lidx, line, print_error=True):
+    e = 0
+    for method in ['_', 'gettext']:
+        for stringtype in ['"', '\[=*\[']:
+            if re.compile(method + ' *' + stringtype).search(line):
+                e = e + 1
+                if print_error:
+                    print('Optional parentheses missing at {}:{}: {}'.format(
+                        filename, lidx, line))
+    return e
+
+
+errors = 0
+
+# Selftests against false-positives
+for line in [
+        '_("Hello World")',
+        'gettext("Hello World")',
+        '_([[Hi]])',
+        '_([[Hi %1%]]):bformat("Foo")',
+]:
```

Comment:
Please indent or better make make an extra variable for the list. It will make the code more readable, imho.

Same for false negatives.